compiler/optimizer/OMRValuePropagation.cpp

```diff
@@ -4395,12 +4442,81 @@ void TR::GlobalValuePropagation::processNaturalLoop(TR_StructureSubGraphNode *no
       collectInductionVariableEntryConstraints();
 
    processRegionSubgraph(node, lastTimeThrough, true, true);
+
```

Comment:
Do exception predecessors also need to be considered here ?

Reply:
No I don't believe so - an expcetion predecessor will mean that we just say that the _definedOnAllPath interest is empty because we assume nothing was defined from that predecessor/